tasks/conftest-clair.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: conftest-clair
+spec:
+  results:
+    - name: HACBS_TEST_OUTPUT
+      description: test output
+  steps:
+    - name: conftest-vulnerabilities
+      image: quay.io/redhat-appstudio/hacbs-test:latest
```

Comment:
I believe we should be using the `quay.io/redhat-appstudio/hacbs-test:stable` for now until we get a release process going.

Reply:
Thanks for notice. Let me change that.